janusgraph-cql/pom.xml

```diff
@@ -311,4 +311,97 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <!-- Run Thrift tests with ScyllaDB (requires Docker) -->
+            <id>scylladb-test</id>
+            <properties>
```

Comment:
can we create a cassandra-base pom to share properties like these? I saw them repeated above